src/growth.jl

```diff
@@ -33,28 +50,54 @@ end
 end
 
 # Rules
+# euler solver rules
+@inline rule(model::EulerExponentialGrowth, data, state, args...) = state + state * model.r
```

Comment:
Just noticed this should be a `state + state * model.r * model.timestep`

Reply:
You can just keep pushing commits fixing things and theyll show up hete

Reply:
But also I don't get the use of `model.timestep` here. Doesn't it means we have an increasing growth rate over time? Isn't exponential growth the outcome of running this multiple times as is?

Reply:
I think ideally, we have a global timestep that is passed to Models(). The rate is dN/dt = r*N so the change in N with each timestep is dN = r*N/dT

Reply:
Yeah I got it after I sent that comment. timestep is timestep length, I was thinking it was t. 

We could add that to the Models and FrameData structs so it's set at the model level and passed to everything, as  cellsize is.